app.js

```diff
@@ -0,0 +1,23 @@
+const express = require("express");
+const app = express();
+app.use(express.json());
+
+const { getTopics } = require("./controllers/topics.controller.js");
+
+app.get("/api/topics", getTopics);
+
+app.use((req, res, next) => {
+    const err = new Error("Path not found");
+    err.status = 404;
+    next(err);
```

Comment:
We can tidy this up a bit if we use app.all(/api/*) which will handle any invalid endpoints that are entered. This would then link to a controller that sends the 404 and message.